Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -154,6 +154,12 @@
             this.gridState.saveGridOptions(this.mainGrid);
         }
 
+        // Resets the scrollbar position
+        private onPaging = () => {
+            this.mainGrid.content.scrollTop(0);
```

Comment:
De her to linjer er de samme overalt. Kan du ikke smide dem i en helper, hvor metodenavnet så hedder "resetScrollbar()". Så skal de samme to linjer ikke følges af den samme kommentar. Så er det selvforklarebde

Reply:
Er blevet lagt i en utility klasse.